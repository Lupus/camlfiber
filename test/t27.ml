let main () =
  let a = Fiber.create (fun () -> print_string "ok";
				  print_newline ()) () in
  print_string "before_wake";
  print_newline ();
  Fiber.wake a;
  Fiber.cancel_wake a;
  print_string "after_wake";
  print_newline ();
  Fiber.resume a

let _ = Fiber.run main ()
