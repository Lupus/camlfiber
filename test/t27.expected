before_wake
after_wake
ok
